pages/manual/planet.md

```diff
@@ -60,7 +60,26 @@ There are several different roles available in Planet: "admin", "manager", "lear
 ### User profile
 Click on the profile button in the top right corner of Planet. Here, you can view or edit your profile, change your password, or logout. Be sure to log out once you are finished with your session.
 
-![](images/planet-home-profile.png)
+![](images/planet-home-profile.png)  
+
+---  
+
+# Using Planet  
+For users that use the nation version of Planet, they will see __Nation__ highlighted in the menu (see first image) and those that use the community version of Planet, they will see __Community__ highlighted in the menu (see second image).
+
+![nation version dashboard](images/nation-page-view.png)
+![community version dashboard](images/community-page-view.png)
+
+For community version users, the following section breaks down the main actions you can do on Planet:
```

Comment:
It is same for nation and community

Reply:
Ok, got it! I have removed the words "for community version users." and merged this branch with the latest upstream/master.